CMakeLists.txt

```diff
@@ -1697,8 +1691,6 @@ install(TARGETS ${_INSTALL_LIBS}
 ##### Export files #####
 if (APPLE)
   set(PKG_PREFIX "SDL2.framework/Resources")
```

Comment:
I guess this line should be fixed too. `lib/cmake/SDL2` will work for all cases.

Reply:
> ${CMAKE_CURRENT_SOURCE_DIR}/SDL2Config.cmake

This should be removed. Config generated from `cmake/Config.cmake.in`.